pkg/network/netbinding/netbinding.go

```diff
@@ -33,6 +33,8 @@ import (
 )
 
 func NetBindingPluginSidecarList(vmi *v1.VirtualMachineInstance, config *v1.KubeVirtConfiguration, recorder k8srecord.EventRecorder) (hooks.HookSidecarList, error) {
+	warnUnregisteredPlugins(vmi, config, recorder)
```

Comment:
I would prefer to do this in two steps:
- Validate plugin registration: Returning an error if the validation failed.
- Record an event when the validation failed (e.g. the error message is sent).

This should also help testing, which seems to be missing from this PR.
I would prefer unit tests if possible (I remember vaguely that there is a fake recorder for the events, but I am not sure).

Reply:
Done